cookbooks/van-keken-vof/doc/van-keken-vof.tex

```diff
@@ -0,0 +1,216 @@
+with fluid~1, and negative in the region with fluid~2.
+However $\tilde{\psi}$ does not satisfy the requirement that the magnitude is
+equal to the distance to the interface as would be required for the signed
+distance function $\tilde{\phi}$.
+The modifications to the parameter file that are necessary in order to draw the reconstructed 
+boundary as a contour are shown immediately below.
+The full configuration file for this version of the benchmark problem can be found at 
+\url{cookbooks/van-keken-vof.prm}.
+
+\lstinputlisting[language=prmfile]{cookbooks/van-keken-vof/doc/postprocess.part.prm.out}
+\noindent
+
+We made a number of computations of the van Keken problem with the VOF method in order to 
+compare the wall clock times with computations using a DG compositional field.
+We ran both on the same cluster at global refinements 5--8 using one node with four CPUs and 
+9--10 using two nodes with 16 CPUs.
+Our results are shown in Table~\ref{tab:vof-runtime-comparison-table}.
+In all of the computations shown in Table~\ref{tab:vof-runtime-comparison-table} we used a CFL 
+number of $\sigma=0.5$.
+Due to the change in the CFL number from $\sigma = 1.0$ in Table~\ref{tab:runtime-table} to 
+$\sigma = 0.5$ in Table~\ref{tab:vof-runtime-comparison-table} and the difference between HPC 
+clusters on which the computational results shown in the two tables were made, we can't make a
+direct quantitative comparison between the data in Tables~\ref{tab:runtime-table} 
+and~\ref{tab:vof-runtime-comparison-table}.
+
+However, we can compare the required run time for a VOF computation to that for a DG computation.
+We note that the use of the VOF advection algorithm significantly reduces the
+required computation time in all cases, frequently requiring less than half the
+time required by the DG compositional field.
+
+We now examine the RMS velocity data shown in Figure~\ref{fig:vof-vk-1}.
+Other than for the case of 5 levels of uniform global refinement, the curves for the RMS 
+velocities for $6$, $7$, $8$, $9$ and $10$ levels of refinement in Figure~\ref{fig:vof-vk-1} 
+are nearly indistinguishable.
+
+Upon examining the solution at the final time, we note that the general structure of the 
+solution shown in Figure~\ref{fig:VOF_van_Keken-02} matches the form and the general structure 
+found in other versions of this benchmark such as the fourth image on the right in 
+Figure~\ref{fig:vk-1}.
+We also note that the differences in the shape of the interface based on a single refinement 
+as shown in Figure~\ref{fig:VOF_van_Keken-02} are minor, although still slightly visible.
+This is to be expected as refinement is a perturbation of the initial condition at a smaller 
+wave length.
+
+\begin{table}
+  \centering
+   \begin{tabular}{|c|c|c|c|}
+     \hline
+      Global Refinement & Number of Processors & VOF & DG \\
+      \hline
+\phantom{1}5 & \phantom{1}4 &\phantom{6}1.33 minutes\phantom{hours} &\phantom{1}2.57 minutes  \\
```

Comment:
You won't need the `\phantom` part in the first two columns if you simply make them right-aligned in the `{|c|c|c|c|}` description above -- but you can also just leave it as is given that it shows nicely in the current version

Reply:
I'm in favor of leaving it as is.  (I wrote it. :-)